src/Tab.js

```diff
@@ -8,7 +8,7 @@ import TabPane from './TabPane';
 /* eslint-disable react/require-render-return, react/no-unused-prop-types */
 class Tab extends React.Component {
   static propTypes = {
-    title: PropTypes.node.isRequired,
+    title: PropTypes.node,
```

Comment:
It seems like requiring this is good given the semantics below, though.

Can we just use `shouldWarn` for that test case below?